profile/ffsn.py

```diff
@@ -0,0 +1,80 @@
+import time
+import numpy as np
+from pyffs import ffsn_sample, ffsn, ffsn_comp
+from pyffs.func import dirichlet_2D
+import matplotlib.pyplot as plt
+import click
+
+
+@click.command()
+@click.option("--n_trials", type=int, default=5)
+def profile_ffsn(n_trials):
+    print("\nCOMPARING FFSN APPROACHES WITH {} TRIALS".format(n_trials))
+
+    T = [1, 1]
+    T_c = [0, 0]
+    N_FS_vals = [101, 301, 1001, 3001]
```

Comment:
Any reason to go for these values?
I would recommend augmenting the benchmark to also compute the FS coefficients by choosing the best `N_s` value possible using `scipy.fftpack.next_fast_len`. (See `czt()` implementation that makes use of this.)

Reply:
I chose these values since they are rather evenly spread on the log scale. I've incorporated `scipy.fftpack.next_fast_len` for the DFT length. Below is the comparison plot with 5 trials averaged per point (need to fix the plotting so that the x-axis value doesn't get cut off).

![ffsn_comparison](https://user-images.githubusercontent.com/4757445/97703301-b0efbd80-1ab0-11eb-9ea1-48991f368980.png)